user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+# Configuring administrator roles
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* category of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* category is divided into 2 sub-categories: *Roles* and *Users* (see [Managing administrators](managing-administrators.md)).
+
+![Administrator roles interface](../assets/users-permissions/administrator_roles.png)
+
+The *Roles* sub-category of *Administration panel* displays all created roles for the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new administrator role (see [Creating a new role](#creating-a-new-role)),
+- delete an administrator role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an administrator role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 3 administrator roles are defined for any Strapi application:
+
+- Author: to be able to create, manage and publish their own content.
+- Editor: to be able to create content, and manage and publish any content.
+- Super Admin: to be able to access all features and settings. This is the role attributed by default to the first administrator at the creation of the Strapi application.
+
+::: warning ATTENTION
+If you use your Strapi application with the Community Edition (see [Pricing and Plans](https://strapi.io/pricing)), your use of the RBAC feature will be limited. Only the 3 default roles are indeed available, as you cannot create more roles and cannot delete the default ones. It is however possible to edit them, but to an extent:
+
+- Configuring permissions in detail is only available for the Enterprise Edition: with the Community Edition, if you have access to a content type, it is automatically a full access with all permissions.
+- You can only configure permissions for the content types, but not for the plugins and settings of the Strapi application.
+- Custom conditions defined for a specific permission are also only available for the Enterprise Edition.
+
+:::
+
+## Creating a new role
```

Comment:
Should we add a EE label?

Reply:
Yes we should, I know we have a gold label but I wonder if we also need to create bronze and silver ones as well as a combined label?

Reply:
You can have a look at PR #158 where I started to create CustomBadges components to deal with this kind of issue.

The next steps would be making it even more customizable (mostly: change background color). 
Would probably have to fork the existing <Badge /> component from VuePress to achieve this, as I can't find a proper way to override the scoped CSS.

Reply:
+1